awx/main/migrations/0165_added_contentsigning_to_project_model.py

```diff
@@ -37,4 +37,13 @@ class Migration(migrations.Migration):
             ),
         ),
         migrations.RunPython(setup_tower_managed_defaults),
+        migrations.RemoveField(
+            model_name='project',
+            name='signature_validation',
+        ),
```

Comment:
this field should never exist after this PR so not Remove just deleted these line